vcap/vcap/device_mapping.py

```diff
@@ -99,11 +99,11 @@ def map_to_openvino_devices():
         be excluded.
 
         Here are the cases:
-        ['CPU:0', 'MYRIAD', ...] => ['CPU:0', 'MYRIAD']
-            If MYRIAD is whitelisted. Else, it will return ["CPU:0"]
-        ['CPU:0', 'HDDL', ...] =>  ['CPU:0', 'HDDL']
-            If HDDL is whitelisted. Else, it will return ["CPU:0"]
-        ['CPU:0'] => ['CPU:0']
+        ['CPU:0', 'MYRIAD', ...] => ["MULTI:CPU,MYRIAD.1,MYRIAD.2,..."]
```

Comment:
Now with map_to_openvino_devices, it will no longer return multiple devices, eg ["CPU", "MYRIAD1", "MYRIAD2"], but rather it will return an OpenVINO style MULTI plugin. So it would return ["MULTI:CPU,MYRIAD1,MYRIAD2"]. 

This means 1 backend for multiple devices. It's very efficient to allow OpenVINO to decide the inferencing bandwidth through the MULTI plugin.

Reply:
Is it possible to use `MULTI` with just one device, like "MULTI:CPU"? If so, the behavior of this method could be simplified to always map to the MULTI device.

Reply:
Well, it's nice to say not MULTI because when you specify an exact device, it's possible to query that device and do specific optimizations for it. When you say MULTI:device, it will put an abstraction layer that prevents you from (easily) getting metrics from sub-devices.